### DISCUSSION TITLE
health problems?

### DISCUSSION THREAD

**Discussion by josef2600** [pschatzmann/arduino-audio-tools#1085] [General]
sorry to ask but, i was reading posts, and i sow you are in hospital again. is it serious ? as i remember it is the second time in few month you are in hospital. would you like some flowers?

**Comment by pschatzmann** [comment]
Actually it is quite bad: at my last operation the diagnose was bladder cancer and that I needed to remove my bladder. This summer and fall I had a chemotherapy as a preparation for the operation which was quite ugly as well.

This Monday, I had finally the Cystectomy and they built me a neobladder.

**Reply by josef2600** [reply]
i hope you get better without more complications.

**Reply by ANKA0991** [reply]
love your work! hope u get better soon!

**Reply by SandeepJagdev** [reply]
get well soon

**Comment by jepsenphil** [comment]
Sending healing vibes your way @pschatzmann.  Love your work, thank you.

**Comment by musiq021** [comment]
Hope your healing goes well Phil!

**Comment by onurceleebi5** [comment]
[Self-harm scar](https://armscars.com/) removal involves various treatments aimed at reducing visibility of scars caused by self-injury. Options include laser therapy, which diminishes scar appearance by stimulating skin regeneration; surgical revision, where scars are surgically altered to be less noticeable; and microdermabrasion, which exfoliates the top skin layer, promoting new, smoother skin growth. Topical treatments like silicone gels or sheets can also be effective, flattening and fading scars over time. It's important to consult with a dermatologist or plastic surgeon to choose the best method for individual cases. Psychological support is equally crucial, as addressing underlying emotional issues can prevent future self-harm and promote overall healing.

**Comment by josef2600** [comment]
welcome back!

**Comment by pschatzmann** [comment]
Thanks, I still have quite some challenges ahead of me, but finally they started to give me food again...

**Comment by willdatrill2007** [comment]
Welcome back.

**Comment by pschatzmann** [comment]
Thanks for all good wishes.

**Reply by josef2600** [reply]
hehe ! he's back!

**Comment by pschatzmann** [comment]
Finally I made it back home today! 

It was a bit depressing to spend Christmas and New Year's eve in a hospital...

**Reply by joba-1** [reply]
so, welcome back! 

Hope you can still celebrate a bit with friends & family and … get well.

**Reply by josef2600** [reply]
welcome back.

**Comment by eccessivo75** [comment]
I am extremely glad to welcome you home. Get some rest, get well.

**Comment by KevynWithaY** [comment]
Welcome home and I would also like to wish you well. Have hope and never give up. I think having a "neobladder" technically makes you a cyborg, which is kinda cool ;)

**Comment by mooballs** [comment]
Blimey, get well soon Phil. Glad you're back home, all the best.
